youtube_dl/extractor/facebook.py

```diff
@@ -451,14 +454,20 @@ def _lowercase_escape(s):
                    self._search_regex(r'ownerName"\s*:\s*"([^"]+)"', webpage, 'uploader', default=None) or \
                    self._og_search_title(webpage, default=None)
 
+        if webpage.find('Paid Partnership'):
+            timestamp = self._search_regex(
+                r'datePublished":"(.+?)"', webpage,
+                'timestamp', default=None)
+            timestamp = parse_iso8601(timestamp)
+        else:
+            timestamp = int_or_none(self._search_regex(
```

Comment:
please change the code so that instead 
AAAAA OR BBB
BBBB OR CCCC
CCCCC etc.
we will have:
AAAAA or
BBBBBB or 
CCCCCC